communications/GSOD-applications/GSoD-2020-Project-Report.md

```diff
@@ -0,0 +1,152 @@
+
+### 3. Improving the Overall Accessibility of the Online Book
+
+**Goal**: *Improving the accessibility of the material by ensuring that the online book has responsive layout for users reading it in different devices or using accessibility apps.*
+
+My work on this goal focused on minimising the overall information loss readers experienced when reading the book with screen readers or on mobile devices. 
+Already, Jupyter Book allows the book's text to reflow to fit all screen sizes and zoom settings. However, images, videos, and tables had to be responsive for _The Turing Way_ to be readable and look great on all devices. 
+I created several issues to help facilitate the migration of image formatting from vanilla Markdown to Markedly Structured Text, which automatically handles responsive styling for images. 
+Some of these issues also encourage contributors to write descriptive ALT text for the book's images that did not have one. 
+This ensures that people using screen readers are able to understand the image's intended message. 
+
+Finally, I wrote custom CSS to create a better layout and responsive styling for tables and videos across the book. 
+Most importantly, I documented the use of custom CSS in a chapter in the book's Community Handbook.
+
+The following GitHub pull requests and issues, and online content were developed to enhance book's accessibility:
+
+| Task | PRs/Issues/online content |
+| -------- | -------- | 
+| Responsive styling for tables     | [#1656](https://github.com/alan-turing-institute/the-turing-way/pull/1656)     |
+|Responsive styling for videos|[#1651](https://github.com/alan-turing-institute/the-turing-way/pull/1651)|
+|Resposive formatting for images|[Issue list](https://github.com/alan-turing-institute/the-turing-way/issues?q=is%3Aopen+is%3Aissue+author%3Apaulowoicho+myst)|
+| **PRs merged into online resources** | [Subchapter in style guide](https://the-turing-way.netlify.app/community-handbook/style/style-more-styling.html), [Subchapter on Stylesheets](https://the-turing-way.netlify.app/community-handbook/style/style-custom-styling.html) |
+
+### Other Contributions
+Beyond my GSoD goals, I contributed to _The Turing Way_ project in other ways.
+(Acronym used PR: Pull Request, OLS: Open Life Science)
+
+1. As part of my mentorship in _The Turing Way_, I had the opportunity to join [Open Life Science (OLS)](https://openlifesci.org) mentoring program to gain practical understanding of open science and leadership that I could integrate in my GSoD project.
+It allowed me to closely work with two members, Neha Moopen (Research Data Manager, TU Delft, The Netherlands) as my project partner and Samuel Guay (PhD researcher, Uni Montreal, Canada) as our OLS mentor, and interact with international participants working on different open science projects.
```

Comment:
I just realised that Neha is from Utrecht not Delft, sorry for the wrong info here. Can you please double check and correct?

Reply:
Thanks for catching that! I've pushed a new commit!